isilon_create_users.sh

```diff
@@ -289,15 +340,19 @@ case "$DIST" in
         [ $? -ne 0 ] && addError "Could not add user impala$CLUSTER_NAME to hive$CLUSTER_NAME group in zone $ZONE"
 
         #Set some varliables to take these special case group assignments to the group file
-        sqp=`grep sqoop$CLUSTER_NAME $grpfile`
-        hve=`grep hive$CLUSTER_NAME $grpfile`
+        sqp=`grep "\bsqoop$CLUSTER_NAME\b": $grpfile`
+        sqp2=`grep "\bsqoop2$CLUSTER_NAME\b": $grpfile`
+        hve=`grep "\bhive$CLUSTER_NAME\b": $grpfile`
 
         #manipulate the group file for special cases
         sed -i .bak /$sqp/d $grpfile
-        echo "$sqp,sqoop2$CLUSTER_NAME" | cat >> $grpfile
+        sed -i .bak /$sqp2/d $grpfile
+        echo "$sqp""sqoop2$CLUSTER_NAME" | cat >> $grpfile
         [ $? -ne 0 ] && addError "Could not add user sqoop2$CLUSTER_NAME to sqoop$CLUSTER_NAME group in $grpfile"
+        echo "$sqp2""sqoop$CLUSTER_NAME" | cat >> $grpfile
+        [ $? -ne 0 ] && addError "Could not add user sqoop$CLUSTER_NAME to sqoop$CLUSTER_NAME group in $grpfile"
```

Comment:
Looks like the message on this guy needs to be updated (i.e. it's trying to add `sqoop` to the `sqoop2` group not the `sqoop` group).